File: app/src/main/java/com/corona/covid_19tracker/Activitys/Fragment/PreventionActivity.java
package com.corona.covid_19tracker.Activitys.Fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;


import androidx.fragment.app.Fragment;

import com.corona.covid_19tracker.R;

public class PreventionActivity extends Fragment {

    @Override
    public View onCreateView( LayoutInflater inflater,  ViewGroup container,  Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.activity_prevention,container,false);
        return view;
    }
}

File: app/src/main/java/com/corona/covid_19tracker/Activity/Fragment/PreventionActivity.kt
package com.corona.covid_19tracker.Activity.Fragment

import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import com.corona.covid_19tracker.databinding.ActivityPreventionBinding
import com.corona.covid_19tracker.utils.AdsTask

class PreventionActivity : Fragment() {
    private lateinit var binding: ActivityPreventionBinding
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = ActivityPreventionBinding.inflate(layoutInflater)


        AdsTask(requireContext()).setBannerAds(binding.adView)


        return binding.root
    }
}